amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpTheme.hbm.xml

```diff
@@ -2,7 +2,6 @@
 <!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 <hibernate-mapping>
 	<class name="org.digijava.module.aim.dbentity.AmpTheme" table="AMP_THEME">
-		<cache usage="read-write" />
```

Comment:
istead of not using cache at all lets evict the theme class after saving or updating a program.
sessionFactory.getCache().evictEntityData(AmpTheme.class);

Reply:
This works on the first save after AMP has started/restarted. It does not work in subsequent saves.

Reply:
it works always. merging